discovery/kubernetes/node.go

```diff
@@ -42,14 +41,15 @@ var (
 
 // Node discovers Kubernetes nodes.
 type Node struct {
-	logger   log.Logger
-	informer cache.SharedInformer
-	store    cache.Store
-	queue    *workqueue.Type
+	logger         log.Logger
+	informer       cache.SharedInformer
+	store          cache.Store
+	queue          *workqueue.Type
+	preferNodeType []string
 }
 
 // NewNode returns a new node discovery.
-func NewNode(l log.Logger, inf cache.SharedInformer) *Node {
+func NewNode(l log.Logger, inf cache.SharedInformer, preferNodeTypes []string) *Node {
 	if l == nil {
 		l = log.NewNopLogger()
```

Comment:
What about making Line 55 multi line and add the prefered address types here?

Reply:
Fine, It would be more readable.